tcms/core/management/commands/refresh_permissions.py

```diff
@@ -20,10 +20,7 @@ def add_arguments(self, parser):
         )
 
     def handle(self, *args, **kwargs):
-        output = None
-        if kwargs["verbosity"]:
-            output = self.stdout
-
+        output = self.stdout if kwargs["verbosity"] else None
```

Comment:
Function Command.handle refactored with the following changes:

- Replace if statement with if expression ([assign-if-exp](https://docs.sourcery.ai/refactorings/assign-if-exp/))